XamarinCommunityToolkit/Behaviors/AttachedNumericValidationBehavior.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors
+{
+    public static class AttachedNumericValidationBehavior
+    {
+        public static readonly BindableProperty AttachBehaviorProperty =
+            BindableProperty.CreateAttached("AttachBehavior", typeof(bool), typeof(AttachedNumericValidationBehavior), false, propertyChanged: OnAttachBehaviorChanged);
+
+        public static bool GetAttachBehavior(BindableObject view) => (bool)view.GetValue(AttachBehaviorProperty);
```

Comment:
In our Converters we settled on having these types of things line break at the arrow, so like this:

```
public static bool GetAttachBehavior(BindableObject view) 
    => (bool)view.GetValue(AttachBehaviorProperty);
```